### PR TITLE
Test: dont read qemu console during boot, adjust timeouts

### PR DESCRIPTION
**Pull request by dperpeet**
extends #2038

### CODE REVIEW

**Comment by dperpeet** [comment]
Remove some indeterminate waiting situations, add and adjust timeouts

File: test/testvm.py [hunk 1]
```diff
@@ -752,45 +750,41 @@ def _cleanup(self):
             print >> sys.stderr, "WARNING: Cleanup failed:", str(value)
 
     def kill(self):
+        # attempt graceful shutdown first
```
Comment (mvollmer) [inline test/testvm.py]: I think we shouldn't do this.  A machine is either shut down cleanly via the `shutdown` method or killed unceremoniously.  It is shutdown when it is in 'maintenance' mode and we want to keep the changes to its file system after we are done with it, such as with vm-install.  After a normal test that doesn't change the image at all, we can just kill qemu.  That ought to be faster and more reliable.

**Comment by mvollmer** [comment]
I have included the FIXUP and the wait_ssh adjustement in #2038.

File: test/testvm.py (hunk 1 above)
Comment (dperpeet) [inline test/testvm.py]: I had to add this because the regular kill leaves zombie qemu sessions on my system...

**Comment by dperpeet** [comment]
replaced by #2074

**Pull request closed without merging**